### DISCUSSION TITLE
Is there any detailed instructions on how to install this package, please - Noobs Guide?

### DISCUSSION THREAD

**Discussion by rangersmyth74** [twbs/bootstrap#35632] [Q&A] [answered]
Hey,

I am new to Linux and I need help with installing this package.  I have it downloaded using git clone, but I am not sure what to do next.

I installed npm 8.3 and tried to install using npm install bootstrap. It did something then asked me to run npm load. I am completely confused! 

There are no instructions to the level I require as a complete noob to install on the youtube or internet, I can't seem to find anything more than the git clone download and npm install bootstrap. 

Being annoyed and a bit frustrated at spending an hour trying to install, this is why I am here asking for help.

I use bootstrap on my desktop Windows PC and I love it, I'd love to get it on Linux. 

Any help would be wonderful.

**Comment by mdo** [comment, marked as answer]
Sorry for missing this—was out on medical leave and never saw it.

For future visitors, yes there are guides! It shouldn't be different than your Windows PC, so not sure what you're running into. 

- Quick start: https://getbootstrap.com/docs/5.2/getting-started/introduction/
- Downloading and package managers: https://getbootstrap.com/docs/5.2/getting-started/download/
- Webpack: https://getbootstrap.com/docs/5.2/getting-started/webpack/
- Parcel: https://getbootstrap.com/docs/5.2/getting-started/parcel/
- npm starter repo: https://github.com/twbs/bootstrap-npm-starter

**Reply by rangersmyth74** [reply]
Thank you so much and I hope you are well and back in 100% health. Again, I appreciate the materials to help. 

Take care and get well soon!
